### PR TITLE
fixed the sign-up redirect, removed some console logs, and finished the logic for the victory page

### PR DESCRIPTION
**Pull request by m-sherrill**
finished the logic for the victory page where if the user has already claimed the item, they will not have the button available to claim it again. 

added a redirect on the sign-up page to redirect to the home page.

### CODE REVIEW

File: client/src/components/SignUpForm.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ const SignUpForm = () => {
       Auth.login(data.createUser.token);
       setSuccessMessage('Welcome!! Your are now signed up!')
       sendForm("service_k0uycid", "template_0jr5hbi", form.current, "NZ0ltP_Q1eOniKe9w")
+      window.location.assign('/')
```
Comment (mjamesd) [inline client/src/components/SignUpForm.js:100]: Not `.replace`?

File: client/src/pages/Start.js [hunk 2]
```diff
@@ -49,17 +52,16 @@ const Start = () => {
...
     if (loading) {
         return <h2 style={{ justifyContent:'center'}}>LOADING.....</h2>
     }
 
+    // Removes duplicates in the cities
     const cities = [...new Map(hunts.map(hunt => [hunt.city, hunt])).values()]
-    console.log(cities)
 
     return (
         <><motion.div
```
Comment (mjamesd) [inline client/src/pages/Start.js:67]: Can we remove the fragment here? (Remove the `<>`?)

**Review by mjamesd — APPROVED** [review]

File: client/src/components/SignUpForm.js (hunk 1 above)
Comment (m-sherrill) [reply client/src/components/SignUpForm.js:100]: Assign was how it was originally in the Auth file. I removed it from their to add the history(-1) to login. I moved the assign from the Auth file to just the signup. Before it applied to login and signup

File: client/src/pages/Start.js (hunk 2 above)
Comment (m-sherrill) [reply client/src/pages/Start.js:67]: I can look but likely no. Sometimes you need the fragment or the whole code block breaks!

**Pull request merged**